### ISSUE TITLE
Support for ruby 3.0.0 ?

### ISSUE THREAD

**Issue by diedthreetimes** [sparklemotion/sqlite3-ruby#307]
Does any version of sqlite-ruby support ruby version 3.0.0?

For at least version 1.3.11, which [claims](https://rubygems.org/gems/sqlite3/versions/1.3.13) support, I get the following error whenever invoking the gem.

```
symbol lookup error: /home/<redacted>/.rvm/gems/ruby-3.0.0/gems/sqlite3-1.3.13/lib/sqlite3/sqlite3_native.so: undefined symbol: rb_check_safe_obj
```

**Comment by mishina2228** [comment]
That seems to be fixed in v1.4.2 which is the latest version. 

https://github.com/sparklemotion/sqlite3-ruby/blob/194493be44cd41b73e612eed69ba49354f0d32e7/CHANGELOG.rdoc#label-1.4.2
>Fix Ruby 2.7 rb_check_safe_obj warnings

**Comment by diedthreetimes** [comment]
These warnings are not related. This issue is specific to ruby 3.0.0, and not present in ruby 2.7.*
